test/server.test.js

```diff
@@ -2505,9 +2505,10 @@ test('uncaughtException should handle thrown undefined literal', function(t) {
         }
     );
 
-    CLIENT.get('/foo', function(err, _, res) {
+    CLIENT.get('/foo', function(err, _, res, data) {
         t.ok(err);
         t.equal(res.statusCode, 500);
+        t.equal(data.message, '');
```

Comment:
I guess we could technically send `'undefined'` over the wire, but don't feel strongly.

Reply:
well if you do `throw undefined`, you have a different problem as well